src/components/GuideTile/GuideTile.js

```diff
@@ -55,7 +54,7 @@ const GuideTile = ({
           }
         `}
       >
-        <NewRelicIcon name={icon} size="2.5rem" />
+        {cloneElement(icon, { size: '2.5rem' })}
```

Comment:
would you be able to explain a little bit more info on `cloneElement` and what the use case is here?